api/app/Traits/Generator/GeneratesNominationDoc.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace App\Traits\Generator;
+
+use App\Enums\TalentNominationSubmitterRelationshipToNominator;
+use App\Models\TalentNominationGroup;
+use PhpOffice\PhpWord\Element\Section;
+
+trait GeneratesNominationDoc
+{
+    protected bool $anonymous;
+
+    use GeneratesDoc;
+
+    /**
+     * Generates general info section for a talent nomination group
+     *
+     * @param  Section  $section  The section to add general info to
+     * @param  TalentNominationGroup  $talentNominationGroup  The talent nomination group
+     */
+    protected function generalInfo(Section $section, TalentNominationGroup $talentNominationGroup)
+    {
+        $this->addLabelText($section, $this->localizeHeading('talent_management_event'), $talentNominationGroup->talentNominationEvent->name[$this->lang]);
+        $this->addLabelText($section, $this->localizeHeading('nominations_opening_date'), $talentNominationGroup->talentNominationEvent->open_date->format('F d, Y'));
+        $this->addLabelText($section, $this->localizeHeading('nominations_closing_date'), $talentNominationGroup->talentNominationEvent->close_date->format('F d, Y'));
+        $this->addLabelText($section, $this->localizeHeading('functional_community'), $talentNominationGroup->talentNominationEvent->community->name[$this->lang]);
+    }
+
+    /**
+     * Generates general info section for a talent nomination group
+     *
+     * @param  Section  $section  The section to add nominee info to
+     * @param  TalentNominationGroup  $talentNominationGroup  The talent nomination group
+     */
+    protected function nominee(Section $section, TalentNominationGroup $talentNominationGroup, $headingRank)
+    {
+
+        $nominatedFor = [];
+        if ($talentNominationGroup->advancement_nomination_count > 0) {
+            array_push($nominatedFor, "{$this->localizeHeading('advancement')} ({$talentNominationGroup->advancement_nomination_count})");
+        }
+        if ($talentNominationGroup->lateral_movement_nomination_count > 0) {
+            array_push($nominatedFor, "{$this->localizeHeading('lateral_movement')} ({$talentNominationGroup->lateral_movement_nomination_count})");
+        }
+        if ($talentNominationGroup->development_programs_nomination_count > 0) {
+            array_push($nominatedFor, "{$this->localizeHeading('development_programs')} ({$talentNominationGroup->development_programs_nomination_count})");
+        }
+
+        $section->addTitle($this->localizeHeading('nominee'), $headingRank);
+        $this->addLabelText($section, $this->localizeHeading('functional_community'), "{$talentNominationGroup->nominee->first_name} {$talentNominationGroup->nominee->last_name}");
+        $this->addLabelText($section, $this->localizeHeading('work_email'), $talentNominationGroup->nominee->work_email);
+        $this->addLabelText($section, $this->localizeHeading('communication_language'), $talentNominationGroup->nominee->preferred_lang);
```

Comment:
I think you'll need to convert this to the full value
Currently: 

<img width="341" height="36" alt="image" src="https://github.com/user-attachments/assets/2973a290-91d8-49cb-b3c2-1d428f66514b" />

Reply:
Nice catch fixed in [Convert communication language to use enum value](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14327/commits/be16fc053d2a94737d5cd4517ac04a032ce73b25)